engine/src/main/java/org/camunda/bpm/engine/impl/pvm/runtime/operation/PvmAtomicOperationCancelActivity.java

```diff
@@ -70,4 +74,16 @@ public boolean isAsync(PvmExecutionImpl execution) {
     return false;
   }
 
+  protected void setDelayedPayloadToNewScope(PvmExecutionImpl execution, CoreModelElement scope) {

```

Comment:
🔧 It would be nicer without code duplication with `PvmAtomicOperationCreateConcurrentExecution` but we could do it later, too.

Reply:
Yes, I will add this to the follow up ticket ( kind of make sense to do it there as well). A new Util class can be created for an example.